### ISSUE TITLE
"Enter path to installed PhoneGap" fails for phonegap-android git repo's after September 6, 2011

### ISSUE THREAD

**Issue by GoogleCodeExporter** [erhanselman/mobile-web-development-with-phonegap#42]
```
The PhoneGap directory structure was changed which breaks the project creation 
wizard for building from github source.

The workaround is to modify your github copy by copying the new directory to 
the old expected example directory like :

cp -rp bin/templates/project/phonegap/templates/project/assets/www example

```

Original issue reported on code.google.com by `paul.beu...@gmail.com` on 29 Sep 2011 at 4:29

**Comment by GoogleCodeExporter** [comment]
```
Proposed for 1.2.5
```

Original comment by `paul.beu...@gmail.com` on 20 Oct 2011 at 12:07
- Added labels: **Milestone-Release1.2.5**

**Comment by GoogleCodeExporter** [comment]
```
Fixed and released in 1.2.5
```

Original comment by `paul.beu...@gmail.com` on 1 Nov 2011 at 8:39
- Changed state: **Fixed**
